app/build.gradle

```diff
@@ -263,7 +263,7 @@ android {
 
     productFlavors {
         jembi {
-            applicationId "org.jembi.appstore"
+            applicationId "org.fdroid.fdroid"
```

Comment:
I think this workaround is going to restrict us in the future, I recommend my third workaround as described in the issue I created here: https://gitlab.com/fdroid/fdroidclient/issues/1180

Reply:
Updated here https://github.com/jembi/fdroidclient/commit/07b7265bf67d7fa3e1b5f4666b30aaa10e966d2c